static/src/stylesheets/base/_type.scss

```diff
@@ -56,7 +56,7 @@ h6 {
 
 
 // SerifText
-.type-5 { @extend %type-5; }
+.type-5 { @extend %type-5 !optional; }
```

Comment:
Why do we need all the optionals? As it's our codebase can't we ensure they're there?